tests/unit/scripting.tcl

```diff
@@ -1,5 +1,5 @@
 foreach is_eval {0 1} {
-foreach script_compatibility_api {server redis} {
+foreach script_compatibility_api {server valkey} {
```

Comment:
If we want to remove all the references of redis in this file. We can update the compatibility code below to do a find replace of server -> redis, and then update all the code in this file to also use server.